qt_dotgraph/src/qt_dotgraph/dot_to_qt.py

```diff
@@ -242,20 +251,21 @@ def dotcode_to_qt_items(self, dotcode, highlight_level, same_label_siblings=Fals
         #graph = pygraphviz.AGraph(string=self._current_dotcode, strict=False, directed=True)
         #graph.layout(prog='dot')
 
-        nodes = self.parse_nodes(graph, highlight_level)
-        edges = self.parse_edges(graph, nodes, highlight_level, same_label_siblings)
+        nodes = self.parse_nodes(graph, highlight_level, scene)
+        edges = self.parse_edges(graph, nodes, highlight_level, scene, same_label_siblings)
         return nodes, edges
 
-    def parse_nodes(self, graph, highlight_level):
+    def parse_nodes(self, graph, highlight_level, scene):
```

Comment:
Can the new argument be an optional keyword argument (for the same compatibility reason mentioned above).

All callers should pass the keyword argument by explicitly passing the keyword.

Same for `parse_edges` below.